book/index.qmd

```diff
@@ -72,38 +83,57 @@ bma
+Finally, anyone who would like to contribute to our ecosystem should read @sec-extending.
 
-Of course, you can also read the book chronologically, cover to cover from start to finish.
-We have marked any section that contains complex technical information with this symbol: \emoji{warning}⚠️. You may wish to skip these sections if you are only interested in basic functionality. Similarly, we have marked sections that are optional, such as deep dives into machine learning theory, with this symbol: \emoji{sparkles}✨. Readers that are interested in the more technical detail will likely want to pay attention to the tables at the end of each chapter that show the relationship between our S3 'sugar' functions and the underlying R6 classes, this is explained in more detail in @sec-introduction.
+Of course, you can also read the book cover to cover from start to finish.
+We have marked any section that contains complex technical information with this symbol: \emoji{warning}⚠️.
+You may wish to skip these sections if you are only interested in basic functionality.
+Similarly, we have marked sections that are optional, such as deep dives into machine learning theory, with this symbol: \emoji{sparkles}✨.
+Readers that are interested in the more technical detail will likely want to pay attention to the tables at the end of each chapter that show the relationship between our S3 'sugar' functions and the underlying R6 classes; this is explained in more detail in @sec-introduction.
 
-This book tries to follow the Diátaxis framework for documentation and so we include tutorials, how-to guides, API references, and explanations. This means that the conclusion of each chapter includes a short reference to the core functions learnt in the chapter, links to relevant posts in the [mlr3gallery](https://mlr-org.com/gallery.html), as well as a few exercises that will cover content learnt in the chapter. You can find the solutions to these exercises in @sec-solutions.
+This book tries to follow the Diátaxis framework for documentation and so we include tutorials, how-to guides, API references, and explanations.
+This means that the conclusion of each chapter includes a short reference to the core functions learnt in the chapter, links to relevant posts in the [mlr3gallery](https://mlr-org.com/gallery.html), and a few exercises that will cover content introduced in the chapter.
+You can find the solutions to these exercises in @sec-solutions.
 
-Finally, if you want to reproduce any of the results in this book, firstly note the seed set at the top of each chapter (which are simply set as the chapter number) and the `sessionInfo` printed in @sec-session-info.
+Finally, if you want to reproduce any of the results in this book, note that the random seed is set as the chapter number and the `sessionInfo` printed in @sec-session-info.
```

Comment:
BTW I know here we state the seed is chapter number and that's an argument to set `echo=FALSE` above but my guess is not everyone will read this which is why I think we should echo